### PR TITLE
🌐 Add French translation for `deployment/docker.md`

### PR DESCRIPTION
**Pull request by rjNemo**
🤗 

Here is the PR to translate the manual deployment.
See the french translation tracking issue [here](https://github.com/tiangolo/fastapi/issues/1972).

Thanks for the reviews

### CODE REVIEW

**Review by yezz123 — APPROVED** [review]

File: docs/fr/docs/deployment/docker.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+RUN pip install fastapi uvicorn
+
+EXPOSE 80
+
+COPY ./app /app
+
+CMD ["uvicorn", "app.main:app", "--host", "0.0.0.0", "--port", "80"]
+```
+
+## Créez le code **FastAPI**.
```
Comment (Smlep) [inline docs/fr/docs/deployment/docker.md:62]:
```suggestion
## Créer le code **FastAPI**.
```

Your other sections names use "infinitif", such as `## Créer un Dockerfile`

File: docs/fr/docs/deployment/docker.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+```console
+$ docker run -d --name mycontainer -p 80:80 myimage
+```
+
+</div>
+
+Vous disposez maintenant d'un serveur FastAPI optimisé dans un conteneur Docker. Réglé automatiquement pour votre 
+serveur actuel (et le nombre de cœurs du CPU).
+
+## Vérifiez
```
Comment (Smlep) [inline docs/fr/docs/deployment/docker.md:124]:
```suggestion
## Vérifier
```

same reason as for the other section name

File: docs/fr/docs/deployment/docker.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+$ docker build -t myimage .
+
+---> 100%
+```
+
+</div>
+
+## Démarrer le conteneur Docker
+
+* Exécuter un conteneur basé sur votre image :
```
Comment (Smlep) [inline docs/fr/docs/deployment/docker.md:111]:
```suggestion
* Exécutez un conteneur basé sur votre image :
```

File: docs/fr/docs/deployment/docker.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+```
+
+</div>
+
+Vous disposez maintenant d'un serveur FastAPI optimisé dans un conteneur Docker. Réglé automatiquement pour votre 
+serveur actuel (et le nombre de cœurs du CPU).
+
+## Vérifiez
+
+Vous devriez pouvoir accéder via l'URL de votre conteneur Docker, par exemple : <a href="http://192.168.99.100/items/5?q=somequery" class="external-link" target="_blank">http://192.168.99.100/items/5?q=somequery</a> ou <a href="http://127.0.0.1/items/5?q=somequery" class="external-link" target="_blank">http://127.0.0.1/items/5?q=somequery</a> (ou équivalent, en utilisant votre hôte Docker).
```
Comment (Smlep) [inline docs/fr/docs/deployment/docker.md:126]:
```suggestion
Vous devriez pouvoir accéder à votre application via l'URL de votre conteneur Docker, par exemple : <a href="http://192.168.99.100/items/5?q=somequery" class="external-link" target="_blank">http://192.168.99.100/items/5?q=somequery</a> ou <a href="http://127.0.0.1/items/5?q=somequery" class="external-link" target="_blank">http://127.0.0.1/items/5?q=somequery</a> (ou équivalent, en utilisant votre hôte Docker).
```

File: docs/fr/docs/deployment/docker.md [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+![ReDoc](https://fastapi.tiangolo.com/img/index/index-02-redoc-simple.png)
+
+## Traefik
+
+<a href="https://traefik.io/" class="external-link" target="_blank">Traefik</a> est un reverse proxy/load balancer 
+haute performance. Il peut faire office de "Proxy de terminaison TLS" (entre autres fonctionnalités).
+
+Il est intégré à Let's Encrypt. Ainsi, il peut gérer toutes les parties HTTPS, y compris l'acquisition et le renouvellement des certificats.
+
+Il est également intégré à Docker. Ainsi, vous pouvez déclarer vos domaines dans les configurations de chaque application et faire en sorte qu'elle lise ces configurations, génère les certificats HTTPS et serve HTTPS à votre application automatiquement, sans nécessiter aucune modification de sa configuration.
```
Comment (Smlep) [inline docs/fr/docs/deployment/docker.md:157]:
```suggestion
Il est également intégré à Docker. Ainsi, vous pouvez déclarer vos domaines dans les configurations de chaque application et faire en sorte qu'elles lisent ces configurations, génèrent les certificats HTTPS et servent via HTTPS à votre application automatiquement, sans nécessiter aucune modification de leurs configurations.
```

File: docs/fr/docs/deployment/docker.md [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+
+Pour configurer un cluster en mode Docker Swarm avec Traefik et la gestion de HTTPS, suivez ce guide :
+
+### <a href="https://medium.com/@tiangolo/docker-swarm-mode-and-traefik-for-a-https-cluster-20328dba6232" class="external-link" target="_blank">Docker Swarm Mode et Traefik pour un cluster HTTPS</a>
+
+### Déployer une application FastAPI
+
+La façon la plus simple de tout mettre en place, serait d'utiliser les [**Générateurs de projet FastAPI**](../project-generation.md){.internal-link target=_blank}.
+
+Il est conçu pour être intégré à ce cluster Docker Swarm avec Traefik et HTTPS décrit ci-dessus.
```
Comment (Smlep) [inline docs/fr/docs/deployment/docker.md:178]:
```suggestion
Le génerateur de projet adéquat est conçu pour être intégré à ce cluster Docker Swarm avec Traefik et HTTPS décrit ci-dessus.
```

**Review by Smlep — CHANGES_REQUESTED** [review]

File: docs/fr/mkdocs.yml [hunk 7]
```diff
@@ -60,6 +60,8 @@ nav:
 - Tutoriel - Guide utilisateur:
   - tutorial/background-tasks.md
 - async.md
+- Deploiement:
```
Comment (Smlep) [inline docs/fr/mkdocs.yml:63]:
```suggestion
- Déploiement:
```

File: docs/fr/docs/deployment/docker.md [hunk 8]
```diff
@@ -0,0 +1,182 @@
...
+
+Si vous utilisez Docker, vous pouvez utiliser l'image Docker officielle :
+
+## <a href="https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker" class="external-link" target="_blank">tiangolo/uvicorn-gunicorn-fastapi</a>
+
+Cette image est dotée d'un mécanisme d'"auto-tuning", de sorte qu'il vous suffit d'ajouter votre code pour obtenir automatiquement des performances très élevées. Et sans faire de sacrifices.
+
+Mais vous pouvez toujours changer et mettre à jour toutes les configurations avec des variables d'environnement ou des fichiers de configuration.
+
+!!! tip
```
Comment (Smlep) [inline docs/fr/docs/deployment/docker.md:20]:
```suggestion
!!! tip "Astuce"
```

File: docs/fr/docs/deployment/docker.md [hunk 9]
```diff
@@ -0,0 +1,182 @@
+# Déployer avec Docker
+
+Dans cette section, vous verrez des instructions et des liens vers des guides pour savoir comment :
+
+* Faire de votre application **FastAPI** une image/conteneur Docker avec une performance maximale. En environ **5 min**.
+* (Optionnellement) comprendre ce que vous, en tant que développeur, devez savoir sur HTTPS.
+* Configurer un cluster en mode Docker Swarm avec HTTPS automatique, même sur un simple serveur à 5 dollars US/mois. En environ **20 min**.
+* Générez et déployez une application **FastAPI** complète, en utilisant votre cluster Docker Swarm, avec HTTPS, etc. En environ **10 min**.
```
Comment (Smlep) [inline docs/fr/docs/deployment/docker.md:8]:
```suggestion
* Générer et déployer une application **FastAPI** complète, en utilisant votre cluster Docker Swarm, avec HTTPS, etc. En environ **10 min**.
```

File: docs/fr/docs/deployment/docker.md [hunk 10]
```diff
@@ -0,0 +1,182 @@
...
+
+<div class="termy">
+
+```console
+$ docker run -d --name mycontainer -p 80:80 myimage
+```
+
+</div>
+
+Vous disposez maintenant d'un serveur FastAPI optimisé dans un conteneur Docker. Réglé automatiquement pour votre
```
Comment (Smlep) [inline docs/fr/docs/deployment/docker.md:121]:
```suggestion
Vous disposez maintenant d'un serveur FastAPI optimisé dans un conteneur Docker. Configuré automatiquement pour votre 
```

File: docs/fr/docs/deployment/docker.md [hunk 11]
```diff
@@ -0,0 +1,182 @@
...
+
+### Déployer une application FastAPI
+
+La façon la plus simple de tout mettre en place, serait d'utiliser les [**Générateurs de projet FastAPI**](../project-generation.md){.internal-link target=_blank}.
+
+Le génerateur de projet adéquat est conçu pour être intégré à ce cluster Docker Swarm avec Traefik et HTTPS décrit ci-dessus.
+
+Vous pouvez générer un projet en 2 min environ.
+
+Le projet généré a des instructions pour le déployer et le faire prend encore 2 min.
```
Comment (Smlep) [inline docs/fr/docs/deployment/docker.md:182]:
```suggestion
Le projet généré a des instructions pour le déployer et le faire prend 2 min de plus.
```

**Review by Smlep — CHANGES_REQUESTED** [review]

**Review by Smlep — APPROVED** [review]

**Comment by tiangolo** [comment]
Awesome, thank you @rjNemo! :cake: 

And thanks for the reviews @Smlep and @yezz123 :coffee:

**Pull request merged**